packages/canary-web-components/src/components/ic-data-table/ic-data-table.tsx

```diff
@@ -195,9 +196,9 @@ export class DataTable {
   @Prop() height?: string;
 
   /**
-   * If `true`, the selected row is highlighted using a background colour.
+   * If `true`, the selected row is highlighted using a background colour. This will display a checkbox column to the left of the table.
    */
-  @Prop() highlightSelectedRow: boolean = true;
+  @Prop() highlightSelectedRow: boolean = false;
```

Comment:
I'm just wondering if `highlightSelectedRow` is still the best name for the prop since it is now doing a lot more than just highlighting the row / changing the background colour. And also therefore whether the prop description should be a bit different (e.g. essentially removing the bit about the background colour)?

This is what I was thinking but just my opinion - happy to discuss :)

```
/**
 * If `true`, a checkbox column will be displayed to the left of the table which allows multiple rows to be selected.
 */
@Prop() rowSelection: boolean = false;
```